src/errorCheck.js

```diff
@@ -0,0 +1,28 @@
+/* eslint-disable max-len */
```

Comment:
Consider removing the `eslint-disable max-len` comment and ensure the lines do not exceed the maximum length specified in your ESLint configuration. This will improve code readability.